bcda/auth/backend.go

```diff
@@ -261,22 +226,51 @@ func getPublicKey() *rsa.PublicKey {
 	return rsaPub
 }
 
-func getJWTClaims(backend *JWTAuthenticationBackend, tokenString string) jwt.MapClaims {
+func (backend *JWTAuthenticationBackend) GetJWTClaims(tokenString string) jwt.MapClaims {
+	token, err := backend.GetJWTToken(tokenString)
+
+	if err != nil {
+		panic(err)
+	}
+
+	if !token.Valid {
+		return nil
+	}
+
+	return token.Claims.(jwt.MapClaims)
+}
+
+func (backend *JWTAuthenticationBackend) GetJWTToken(tokenString string) (*jwt.Token, error) {
```

Comment:
Not a big deal at all, but can we call this `GetJWT` or `GetToken`, since the T in JWT = Token?

Reply:
getToken isn't going to work because that is ambiguous about what kind of token, our token object in the db vs a JWT token.  I think getJWT is also unclear because that could just be the JWT package or some JWT context instead of an actual JWT token.  JWT is a clear part of our system and has other objects like claims (See line 229) that we use.  I think getJWToken might be fine, but it isn't jwt.oken, it is jwt.Token.

Reply:
Committed a change to GetJWToken